activerecord/test/models/season.rb

```diff
@@ -0,0 +1,3 @@
+class Season < ActiveRecord::Base
+  self.primary_key = nil
+end
```

Comment:
FWIW, there are a few tables that have no primary key. Most are join tables, but `edges` with the `Edge` model may be a good choice. It is even enumerated with the `edge_1` to `edge_4` too.